recipes/cobra-meta/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "cobra-meta" %}
+{% set version = "1.2.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/linxingchen/cobra/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 26f3cab513f8bb78d0aeb5ccfa3aed57bafb62f42bee1ac814e74e43c330a390
+
+build:
+  entry_points:
+    - cobra-meta = cobra:main
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('cobra-meta', max_pin="x") }}
+
+requirements:
+  host:
+    - python
+    - pip
+    - Cython
+  run:
+    - python
+    - biopython
+    - pysam
+    - numpy
+    - Cython
```

```suggestion
    - python >=3.7
    - pip
  run:
    - python >=3.7
    - biopython
    - pysam
    - numpy
```
Please pin the python versions in the `host` and `run` section as the package is listed as `noarch: python`. Also, is `cython` a dependency? I do not see it listed in `setup.py` or in the documentation.